mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulHome.kt

```diff
@@ -82,7 +82,7 @@ fun StatefulHome(
       }
       composable(PlayRoute) {
         StatefulPlayScreen(
-            { controller.navigate(PrepareGameRoute) }, Modifier.fillMaxSize(), paddingValues)
+            user, { controller.navigate(PrepareGameRoute) }, Modifier.fillMaxSize(), paddingValues)
```

Comment:
I know it wasn't previously named, but I think naming arguments would make it much easier to understand what the lambda is for.